{
   "CVE_data_meta" : {
      "ASSIGNER" : "cve@mitre.org",
      "ID" : "CVE-2019-6446",
      "STATE" : "PUBLIC"
   },
   "affects" : {
      "vendor" : {
         "vendor_data" : [
            {
               "product" : {
                  "product_data" : [
                     {
                        "product_name" : "n/a",
                        "version" : {
                           "version_data" : [
                              {
                                 "version_value" : "n/a"
                              }
                           ]
                        }
                     }
                  ]
               },
               "vendor_name" : "n/a"
            }
         ]
      }
   },
   "data_format" : "MITRE",
   "data_type" : "CVE",
   "data_version" : "4.0",
   "description" : {
      "description_data" : [
         {
            "lang" : "eng",
            "value" : "An issue was discovered in NumPy 1.16.0 and earlier. It uses the pickle Python module unsafely, which allows remote attackers to execute arbitrary code via a crafted serialized object, as demonstrated by a numpy.load call."
         }
      ]
   },
   "problemtype" : {
      "problemtype_data" : [
         {
            "description" : [
               {
                  "lang" : "eng",
                  "value" : "n/a"
               }
            ]
         }
      ]
   },
   "references" : {
      "reference_data" : [
         {
            "name" : "https://bugzilla.suse.com/show_bug.cgi?id=1122208",
            "refsource" : "MISC",
            "url" : "https://bugzilla.suse.com/show_bug.cgi?id=1122208"
         },
         {
            "name" : "https://github.com/numpy/numpy/issues/12759",
            "refsource" : "MISC",
            "url" : "https://github.com/numpy/numpy/issues/12759"
         },
         {
            "name" : "106670",
            "refsource" : "BID",
            "url" : "http://www.securityfocus.com/bid/106670"
         }
      ]
   }
}
